### PR TITLE
docker: Update cachalot version to v5

### PR DESCRIPTION
**Pull request by leandro-lucarella-sociomantic**
TODO: Update to the final v4!

### CODE REVIEW

File: beaver.Dockerfile [hunk 1]
```diff
@@ -1 +1 @@
-FROM sociomantictsunami/dlang:v3
+FROM sociomantictsunami/dlang:v5-test
```
Comment (mihails-strasuns-sociomantic) [inline beaver.Dockerfile]: still test?

Comment (leandro-lucarella-sociomantic) [reply beaver.Dockerfile]: nope, I should update. :+1:

Comment (leandro-lucarella-sociomantic) [reply beaver.Dockerfile]: But I'm rebuilding cachalot v5 image with latest dmd1 release, so please don't merge yet.

**Comment by leandro-lucarella-sociomantic** [comment]
Now rebuilding with the final cachalot image, if it's green we are good to merge.

**Pull request merged**